Luminary099/R30.agc

```diff
@@ -246,7 +247,7 @@ DOTICK		CAF	1SEC		# RE-REQUEST TICKTEST.
 		MASK	V82FLAGS
```

Comment:
![image](https://user-images.githubusercontent.com/3440094/95388683-9596f580-08ea-11eb-9bf6-c4a6de5b52a0.png)
Line 236 (above):

- No space before `CANT`
- `CANT` instead of `CAN'T`